src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -82,6 +83,42 @@ public FeedbackSessionAttributes getFeedbackSession(String courseId, String feed
         return feedbackSession;
     }
 
+    /**
+     * Gets a list of feedback sessions within the given time range.
+     */
+    public List<FeedbackSessionAttributes> getAllFeedbackSeesionWithinTimeRange(Instant rangeStart, Instant rangeEnd) {
```

Comment:
I think `getFeedbackSessionsWithinTimeRange` would suffice.

You can briefly add the details of the "union" stuff in the javadocs?